transports/ssl/src/main/java/org/mule/transport/ssl/revocation/StandardRevocationCheck.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.transport.ssl.revocation;
+
+import org.mule.api.security.tls.RevocationCheck;
+
+import java.security.GeneralSecurityException;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertPathBuilder;
+import java.security.cert.PKIXBuilderParameters;
+import java.security.cert.PKIXRevocationChecker;
+import java.security.cert.X509CertSelector;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.net.ssl.CertPathTrustManagerParameters;
+import javax.net.ssl.ManagerFactoryParameters;
+
+/**
+ * Uses the standard JVM certificate revocation checks, which depend on the certificate having the
+ * corresponding extension points (additional tags for CRLDP or OCSP), and the availability
```

Comment:
Can you add the specific names? Might save some trouble when analysing certificates used.